lib/watir/browser.rb

```diff
@@ -72,7 +72,7 @@ def inspect
     #
 
     def goto(uri)
-      uri = "http://#{uri}" unless uri =~ URI.regexp
+      uri = "http://#{uri}" unless uri =~ URI.regexp(%w(http https))
```

Comment:
This gives errors when trying to use:
`about:blank` and `data:text/html;content-type=utf-8,foobar`